jetstream/tests/integration/test_analysis_integration.py

```diff
@@ -538,6 +547,9 @@ def test_logging(self, monkeypatch, client, project_id, static_dataset, temporar
 
         assert len(logs) >= 1
         error_logs = [log for log in logs if log.get("log_level") == "ERROR"]
+        error_logs.sort(key=lambda k: (k.get("segment"), k.get("analysis_basis")))
```

Comment:
I guess that will fix the flaky tests?

Reply:
Yea, that is the goal. My thought was that the cause was an unreliable order of logs.